requirements.txt

```diff
@@ -5,55 +5,87 @@ aiosignal==1.3.2
 annotated-types==0.7.0
 anyio==4.7.0
 attrs==24.3.0
+cachetools==5.5.0
```

Comment:
🔧 Consider verifying if `cachetools==5.5.0` is necessary for your project. If not used, it can be removed to keep dependencies minimal.